### ISSUE TITLE
What features are good candidates for being dropped before adopting a "don't break userspace" (no back-incompat) policy?

### ISSUE THREAD

**Issue by gobengo** [solid/solid-spec#146]
Inspired by tangential discussion in #116.

<table>
  <thead><tr>
    <th>feature</th>
    <th>supported by</th>
  </tr></thead>
  <tr>
    <td><a href="https://github.com/solid/solid-spec/blob/master/api-rest.md#globbing-inlining-on-get">globbing</a></td>
    <td>
      <a href="https://github.com/solid/solid/issues/116">elf-pavlik</a>,
      <a href="https://github.com/solid/solid-spec/issues/145">melvincarvalho</a>
    </td>
  </tr>
</table>

**Comment by elf-pavlik** [comment]
I think [Websockets API](https://github.com/solid/solid-spec/blob/master/api-websockets.md) might need to get evaluated against alternatives / complementary mechanisms like:
* Server-sent Events - https://html.spec.whatwg.org/multipage/server-sent-events.html#server-sent-events
* WebSub - https://www.w3.org/TR/websub/

In general having implementation based feedback for each feature would come very helpful.

**Comment by kjetilk** [comment]
Yeah, I've been trying to ask around for use cases that can distinguish HTTP/2+SSE vs WebSockets, but there hasn't been anything very clear so far.

**Comment by elf-pavlik** [comment]
One more I can think of - WebID-TLS could get limited only as one of the availaable auth method with some OPs, I haven't tested that but if somone has client cert installed in a browser and each RS tries to initiate WebID-TLS session it might cause a browser cert selection pop-up for each of those servers even if someone uses WebID-OIDC to authenticate with them. I could try testing it in next days unless someone already checked that scenario.

**Comment by kjetilk** [comment]
I admit to like WebID-TLS, and I am hoping it can stay, and I see it as a simpler alternative for IoT devices that need to send data to Pods. So, I'm trying to think about architectures that can better support several authns side by side, as I also see several other mechanisms coming sliding in e.g. SAFE and then we have the whole DID, so I think we can't be very restrictive in that space.

**Comment by elf-pavlik** [comment]
Another issue related to WebID-TLS https://github.com/solid/solid-spec/issues/145#issuecomment-480980113

> NSS currently still allows authentication with client certificates. For that, NSS has to terminate the HTTPS connection, and NSS only does HTTP 1.1. While you can put a reverse proxy with HTTP/2 in front of NSS (which is what I do), this does break client-side certificates (or you have to find a way to forward the client certificate negotiation).

**Comment by Ryuno-Ki** [comment]
WebSub would be cool!
(I'm currently getting my feed wet with WebMentions)
